### DISCUSSION TITLE
When free block coding?

### DISCUSSION THREAD

**Discussion by Alex24545** [pybricks/support#1311] [General]
Pybricks just have block coding,which is great!But it's supporters only,why?Why dont let us try blocks for free?

**Comment by laurensvalk** [comment]
Pybricks is a huge project made possible by a very small team.

By supporting Pybricks, you help make it possible for us to work on new features, like block coding. Thank you!

**Reply by Alex24545** [reply]
But then when block coding will be available to everyone?

**Reply by rbeckett75** [reply]
I'd also like to find out when you plan on having blocks available to everyone -Hoping sometime next year?

I provide AFOL/technical support to a lot of fellow builders in my area, and I always point them to pybricks, as a great way to get mechanisms to work in their MOC for exhibits, without needing a tablet.  Most of the time, it's a simple autonomous rotation, or back and forth movement... Blocks will let these beginners-in-coding do more advanced movements!

I'm very happy to support pybricks, to get new features like block coding (as well as nice responses to questions on the forums here!)
as I can afford it, and I think it's worth the money..

**Reply by laurensvalk** [reply]
Thanks @rbeckett75! I appreciate your comments and of course your support of the project.

Note that anyone can open existing block projects, so you can still share your projects with others. Python coding and all other existing features remain freely accessible as well, and open source too.

At the moment, we're planning to continue having block coding as a feature exclusively to reward supporters of the project, so that we can keep working on exciting new features.

Another option would be to find a party who wants to help make it free for everyone. If you find that Pybricks adds value to the LEGO play experience, perhaps consider writing to LEGO to let them know :)
